### PR TITLE
announce pDAI

### PR DESCRIPTION
**Pull request by johannbarbie**
resolves: https://github.com/leapdao/meta/issues/70

### CODE REVIEW

File: src/blog/content/DAI-enabled-as-Plasma-Asset.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: 'DAI enabled as Plasma Asset'
+date: 2019-02-25 16:23:00
+emoji: ✅
+author:
+  - name: '@kohola'
+    twitter: 'koholaa'
+description: DAI stablecoin registered on the Leap Plasma chain
+---
+
+DAI, the first decentralized stablecoin on the Ethereum blockchain has been a real game changer for the Ethereum ecosystem, especially in the midst of ETH price crash. DAI is pegged to the value of USD which means that every DAI is worth $1, regardless of how much DAI is in existence. Moreover, due to the fact that DAI is backed by Ether and locked in smart contracts called Collateralized Debt Positions (CDP), it cannot be shut down like other centralized stable coins, and can be freely traded and exchanged without middlemen like any other ERC20 token.
```
Comment (troggy) [inline src/blog/content/DAI-enabled-as-Plasma-Asset.md]:
```suggestion
DAI, the first decentralized stablecoin on the Ethereum blockchain has been a real game changer for the Ethereum ecosystem, especially in the midst of ETH price crash. DAI is pegged to the value of USD which means that every DAI is worth $1, regardless of how much DAI is in existence. Moreover, due to the fact that DAI is backed by Ether locked in smart contracts called Collateralized Debt Positions (CDP), it cannot be shut down like other centralized stable coins, and can be freely traded and exchanged without middlemen like any other ERC20 token. 
```

Comment (sunify) [inline src/blog/content/DAI-enabled-as-Plasma-Asset.md]: First paragraph way too big
image

File: src/blog/content/DAI-enabled-as-Plasma-Asset.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+The Ethereum community could experience the potential of DAI first hand during ETHDenver, thanks to a simple xDai wallet created by [Austin Griffith](https://github.com/austintgriffith/burner-wallet), director of research at [Gitcoin](https://gitcoin.co/). Every conference attendee received their own unique web wallet pre-loaded with a localcoin called buffiDai, pegged to DAI, and could immediately start participating in this transient crypto-economy by, for example, purchasing their hotdogs with buffDAI. The xDAI chain has demonstrated how fast and cheap transactions are an essential building block of cryptocurrency adoption.
+
+Being big fans of DAI token ourselves, last week we enabled running DAI on Plasma chain, and the [first plasma DAI transaction occurred on our mainnet](https://twitter.com/leapdao/status/1105791584531087360).
+
+<img src="/img/blog/daiTx.png" alt="Plasma Roadmap">
```
Comment (sunify) [inline src/blog/content/DAI-enabled-as-Plasma-Asset.md]: Missed image

File: src/blog/content/DAI-enabled-as-Plasma-Asset.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+template: post.html
+title: 'DAI enabled as Plasma Asset'
+date: 2019-02-25 16:23:00
```
Comment (sunify) [inline src/blog/content/DAI-enabled-as-Plasma-Asset.md]: Wrong date?

**Pull request merged**